translator/trace/zipkin/zipkinv2_to_traces.go

```diff
@@ -343,8 +343,20 @@ func tagsToAttributeMap(tags map[string]string, dest pdata.AttributeMap) error {
 		if _, ok := nonSpanAttributes[key]; ok {
 			continue
 		}
-		dest.UpsertString(key, val)
-		// TODO add translation to native OTLP types if configured to do so
+
+		switch tracetranslator.DetermineValueType(val, false) {
```

Comment:
Should be noted that the v1 conversion code is currently parsing strings https://github.com/open-telemetry/opentelemetry-collector/blob/master/translator/trace/zipkin/zipkinv1_to_protospan.go#L256 and that these should be consistent